doc/calculator/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "calculator"
-version = "0.20.0"
+version = "0.20.1"
```

Comment:
Can't we use something like:

```
version.workspace = true
```

if this crate always follows the version of the main package of the workspace? This would save future version bumps.

Reply:
That's a fair point. It doesn't seem like any of these examples get rendered in the lalrpop book so it seems fine.

Actually on that point, maybe we should be pointing to these examples in the book instead of the tests when possible(referring to http://lalrpop.github.io/lalrpop/tutorial/index.html). Otherwise, I'm not sure how much visibility some of these get.

Reply:
I've spawned this off to it's own pr to be discussed/merged separately.